src/main/java/com/github/xetorthio/jedisque/Job.java

```diff
@@ -1,15 +1,17 @@
 package com.github.xetorthio.jedisque;
 
+import redis.clients.util.SafeEncoder;
+
 public class Job {
 
 	private String id;
 	private String queueName;
-	private String body;
+	private byte[] body;
 
 	public Job() {
 	}
 
-	public Job(String queueName, String id, String body) {
+	public Job(String queueName, String id, byte[] body) {
```

Comment:
And we need to change this constructor to accept `byte[]` to support the comment below.